### DISCUSSION TITLE
How to copy files from nvim tree across multiple nvim instances?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#2144] [Q&A] [answered]
I recently moved to LazyVim, and sometimes I need to copy files/folders between projects. When I copy files to a clipboard it pastes successfully in the same nvim instance but not across multiple ones. It would be a useful feature if I were able to copy folders from one project to another using nvim. Is there an option to enable copy-pasting across different projects?

**Comment by dpetka2001** [comment, marked as answer]
This depends on `neo-tree` itself and which clipboard/register it uses to store the copied information. If it's able to support it, then good. Otherwise just use another file manager for your purposes. Or use the terminal to move files.

**Comment by Demianeen** [comment]
Understood @dpetka2001. 
As far as I know, it supports copying files to the system clipboard. Lazyvim just has custom keymaps. I probably would need to add some Y keymap to add this functionality

**Reply by dpetka2001** [reply]
As far as I can tell it uses an internal clipboard to store saved state of nodes. I might be wrong about that, but I believe it would be better to create an issue at `neo-tree` repo, as they are far more knowledgeable about this. If they come up with a solution I guess it can be incorporated in LazyVim.

**Comment by AdrianDiazCode** [comment]
So how is this done? why is this closed?
